en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md

```diff
@@ -401,6 +428,10 @@ Follow the steps given below:
     {"Hello":"World"}%
     ```
 
+!!! Tip
+    The `ei-operator-ingress` will not be created if you have [disabled the ingress controller](#Disable-ingress-controller-creation).
```

Comment:
ei-operator-ingress?
Do we still use 'ei'?

Reply:
Here, separate ingress is created for only integration based deployments by the operator. Ingress creation can be customized via ConfigMap. However, I suggest **integration-ingress** instead of `ei-operator-ingress` here. 
@nilminiwso2 @abeykoon WDYT?

Reply:
As this needs a code change, will address in a different PR.